python/lsst/ip/isr/vignette.py

```diff
@@ -177,6 +177,7 @@ def maskVignettedRegion(exposure, polygon, maskPlane="NO_DATA", vignetteValue=No
     log = log if log else logging.getLogger(__name__)
     if not polygon:
         log.info("No polygon provided.  Masking nothing.")
+        return
```

Comment:
Can't comment on the line above, but can you also fix the docstring on line 161 which has an extra `.`?